### PR TITLE
TDW-2161 Enable GTM and remove adobe script

### PR DESCRIPTION
**Pull request by VincentLeeEnsemble**
https://pgatour.atlassian.net/browse/TDW-2161

Test URLs:
https://tdw-2161-enable-gtm--pgatour-com--hlxsites.hlx.page/

### CODE REVIEW

File: pages/scripts/scripts.js [hunk 1]
```diff
@@ -199,36 +199,59 @@ function getPageName(sectionName) {
...
 
-export function pushOneTrustConsentGroups() {
-  window.adobeDataLayer = window.adobeDataLayer || [];
-  const dl = window.adobeDataLayer;
-  dl.push({
-    event: 'LaunchOTLoaded',
-    // eslint-disable-next-line no-undef
-    OnetrustActiveGroups: typeof OnetrustActiveGroups !== 'undefined' ? OnetrustActiveGroups : '',
-  });
+export async function fetchPlaceholders(prefix = 'default') {
```
Comment (Mark-M-Ensemble) [inline pages/scripts/scripts.js:211]: Why was this added to scripts?

This is already in lib-franklin, we can import it from there.

File: pages/scripts/scripts.js [hunk 2]
```diff
@@ -199,36 +199,59 @@ function getPageName(sectionName) {
...
 }
 
-export function clearDataLayer() {
-  window.adobeDataLayer = [];
+/*
+ * Sanitizes a name for use as a js property name.
+ * @param {string} name The unsanitized name
+ * @returns {string} The camelCased name
+ */
+export function toCamelCase(name) {
```
Comment (Mark-M-Ensemble) [inline pages/scripts/scripts.js:207]: Same for this one, we can import from lib-franklin

**Review by Mark-M-Ensemble — COMMENTED** [review]
Some functions that could be imported.

File: pages/scripts/scripts.js (hunk 1 above)
Comment (VincentLeeEnsemble) [reply pages/scripts/scripts.js:211]: oh, in pga-mitsubishi-electric-championship, it was added in the scripts file. I will import it from lib-franklin

File: pages/scripts/delayed.js [hunk 3]
```diff
@@ -86,47 +84,32 @@ async function OptanonWrapper() {
     localStorage.setItem('OptIn_PreviousPermissions', JSON.stringify(adobeSettings));
   }
 
-  clearDataLayer();
-  loadScript(`https://assets.adobedtm.com/d17bac9530d5/a14f7717d75d/launch-aa66aad171be${isProd ? '.min' : ''}.js`, () => {
-    pushOneTrustConsentGroups();
-
-    loadScript('https://cdns.us1.gigya.com/js/gigya.js?apikey=3_IscKmAoYcuwP8zpTnatC3hXBUm8rPuI-Hg_cZJ-jL-M7LgqCkxmwe-ps1Qy7PoWd', () => {
```
Comment (carolinekennedy-pgatour) [inline pages/scripts/delayed.js:93]: I think this script might need to stay. I think it should be moved out the adobe loadscript

**Review by Mark-M-Ensemble — APPROVED** [review]

File: pages/scripts/delayed.js [hunk 4]
```diff
@@ -81,41 +78,17 @@ async function OptanonWrapper() {
...
-    const adobeSettings = OneTrustActiveGroup();
-    adobeSettings.tempImplied = true;
-    localStorage.setItem('OptIn_PreviousPermissions', JSON.stringify(adobeSettings));
+    const settings = OneTrustActiveGroup();
+    settings.tempImplied = true;
+    localStorage.setItem('OptIn_PreviousPermissions', JSON.stringify(settings));
   }
 
-  clearDataLayer();
-  loadScript(`https://assets.adobedtm.com/d17bac9530d5/a14f7717d75d/launch-aa66aad171be${isProd ? '.min' : ''}.js`, () => {
```
Comment (Mark-M-Ensemble) [inline pages/scripts/delayed.js:90]: Looks like they called Gigya after launch was added previously.

Is there any reason we should do this also for GTM?

File: pages/scripts/delayed.js [hunk 5]
```diff
@@ -124,9 +97,25 @@ const otId = placeholders.onetrustId;
 if (otId) {
   const cookieScript = loadScript('https://cdn.cookielaw.org/scripttemplates/otSDKStub.js');
   cookieScript.setAttribute('data-domain-script', `${otId}${isProd ? '' : '-test'}`);
-  cookieScript.setAttribute('data-dlayer-name', 'adobeDataLayer');
+  cookieScript.setAttribute('data-dlayer-name', 'dataLayer');
```
Comment (Mark-M-Ensemble) [inline pages/scripts/delayed.js:100]: Is this needed for something if we no longer push to the dataLayer?

**Review by Mark-M-Ensemble — APPROVED** [review]

File: pages/scripts/delayed.js (hunk 4 above)
Comment (Mark-M-Ensemble) [reply pages/scripts/delayed.js:90]: This was done to pass user info in the now removed sendPageEvents()

File: pages/scripts/delayed.js (hunk 5 above)
Comment (VincentLeeEnsemble) [reply pages/scripts/delayed.js:100]: Probably not, but I am not sure

File: pages/scripts/delayed.js (hunk 4 above)
Comment (VincentLeeEnsemble) [reply pages/scripts/delayed.js:90]: Removed, we don't need the gigya id in the datalayer.
